annotations/src/main/java/io/quarkiverse/operatorsdk/annotations/RBACVerbs.java

```diff
@@ -12,6 +12,7 @@ public class RBACVerbs {
     public static final String WATCH = "watch";
     public static final String DELETE = "delete";
     public static final String[] UPDATE_VERBS = new String[] { PATCH, UPDATE };
+    public static final String[] CREATE_VERBS = new String[] { CREATE, PATCH };
```

Comment:
PATCH should create if not exist? It just seems a little off to me.

Reply:
I don't recall why we added patch there but I seem to recall it might be required when you use SSA.

Reply:
That said, you're right, PATCH should only be added if SSA is used.